File: engine/engine-model/src/main/scala/net/server/channel/packet/bbs/ReplyToThreadPacket.scala
package net.server.channel.packet.bbs

class ReplyToThreadPacket(private var _mode: Byte, private var _threadId: Int, private var _message: String) extends BaseBBSOperationPacket(_mode = _mode) {
  def threadId: Int = _threadId

  def message: String = _message
}

File: engine/engine-model/src/main/java/net/server/channel/packet/bbs/ReplyToThreadPacket.java
package net.server.channel.packet.bbs;

public class ReplyToThreadPacket extends BaseBBSOperationPacket {
   private final Integer threadId;

   private final String message;

   public ReplyToThreadPacket(Byte mode, Integer threadId, String message) {
      super(mode);
      this.threadId = threadId;
      this.message = message;
   }

   public Integer threadId() {
      return threadId;
   }

   public String message() {
      return message;
   }
}
